cores/cv32e40p/user_manual/source/core_versions.rst

```diff
@@ -0,0 +1,91 @@
+Core Versions and RTL Freeze Rules
+==================================
+
+The CV32E40P is defined by the ``marchid`` and ``mimpid`` tuple.
+The tuple identify which sets of parameters have been verified
+by OpenHW Group, and once RTL Freeze is achieved, no further
+non-logically equivalent changes are allowed on that set of parameters.
+
+The RTL Freeze version of the core is indentified by a GitHub
+tag with the format cv32e40p_vMAJOR.MINOR.PATCH (e.g. cv32e40p_v1.0.0).
+In addition, the release date is reported in the documentation.
+
+What happens after RTL Freeze?
+------------------------------
+
+A bug is found
+^^^^^^^^^^^^^^
+
+If a bug is found that affect the already frozen parameter set,
+the RTL changes required to fix such bug are non-logically equivalent by definition.
+Therefore, the RTL changes are applied only on a different  ``mimpid``
+value and the bug and the fix must be documented.
+These changes are visible by software as the ``mimpid`` has a different value.
+Every bug found must be followed by another RTL Freeze release and a new GitHub tag.
+
+RTL changes on non-verified yet parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If changes affecting the core on a non-frozen parameter set are required,
+as for example, to fix bugs found in the communication to the FPU (e.g., affecting the core only if ``FPU=1``),
+or to change the ISA Extensions decoding of PULP instructions (e.g., affecting the core only if ``PULP_XPULP=1``),
+then such changes must be logically equivalent to the already frozen set of parameters, and they must be applied on a different ``mimpid`` value. They can be non-logically equivalent to a non-frozen set of parameters.
```

Comment:
then such changes remain logically equivalency for the already frozen set of parameters (except for the required mimpid update)